helpers/loginBlock.js

```diff
@@ -0,0 +1,9 @@
+// check if user is logged in
+module.exports = function(req, res, next) {
```

Comment:
at some codes you use `exports.<fn-name> = function() {}`

at some codes you use `module.exports`

choose one way and stick to one.